api/datadoghq/common/envvar.go

```diff
@@ -5,194 +5,77 @@
-	DDRuntimeSecurityConfigRemoteConfigurationEnabled    = "DD_RUNTIME_SECURITY_CONFIG_REMOTE_CONFIGURATION_ENABLED"
-	DDContainerImageEnabled                              = "DD_CONTAINER_IMAGE_ENABLED"
-	DDSBOMEnabled                                        = "DD_SBOM_ENABLED"
-	DDSBOMContainerImageEnabled                          = "DD_SBOM_CONTAINER_IMAGE_ENABLED"
-	DDSBOMContainerImageAnalyzers                        = "DD_SBOM_CONTAINER_IMAGE_ANALYZERS"
-	DDSBOMContainerUseMount                              = "DD_SBOM_CONTAINER_IMAGE_USE_MOUNT"
-	DDSBOMContainerOverlayFSDirectScan                   = "DD_SBOM_CONTAINER_IMAGE_OVERLAYFS_DIRECT_SCAN"
-	DDSBOMHostEnabled                                    = "DD_SBOM_HOST_ENABLED"
-	DDSBOMHostAnalyzers                                  = "DD_SBOM_HOST_ANALYZERS"
-	DDSecretBackendCommand                               = "DD_SECRET_BACKEND_COMMAND"
-	DDSecretBackendArguments                             = "DD_SECRET_BACKEND_ARGUMENTS"
-	DDSecretBackendTimeout                               = "DD_SECRET_BACKEND_TIMEOUT"
-	DDServiceDiscoveryEnabled                            = "DD_DISCOVERY_ENABLED"
-	DDSite                                               = "DD_SITE"
-	DDSystemProbeAgentEnabled                            = "DD_SYSTEM_PROBE_ENABLED"
-	DDSystemProbeBPFDebugEnabled                         = DDSystemProbeEnvPrefix + "BPF_DEBUG"
-	DDSystemProbeCollectDNSStatsEnabled                  = "DD_COLLECT_DNS_STATS"
-	DDSystemProbeConntrackEnabled                        = DDSystemProbeEnvPrefix + "ENABLE_CONNTRACK"
-	DDSystemProbeDebugPort                               = DDSystemProbeEnvPrefix + "DEBUG_PORT"
-	DDSystemProbeEnabled                                 = "DD_SYSTEM_PROBE_ENABLED"
-	DDSystemProbeEnvPrefix                               = "DD_SYSTEM_PROBE_CONFIG_"
-	DDSystemProbeExternal                                = "DD_SYSTEM_PROBE_EXTERNAL"
-	DDSystemProbeNPMEnabled                              = "DD_SYSTEM_PROBE_NETWORK_ENABLED"
-	DDSystemProbeServiceMonitoringEnabled                = "DD_SYSTEM_PROBE_SERVICE_MONITORING_ENABLED"
-	DDSystemProbeSocket                                  = "DD_SYSPROBE_SOCKET"
-	DDSystemProbeOOMKillEnabled                          = DDSystemProbeEnvPrefix + "ENABLE_OOM_KILL"
-	DDSystemProbeTCPQueueLengthEnabled                   = DDSystemProbeEnvPrefix + "ENABLE_TCP_QUEUE_LENGTH"
-	DDTags                                               = "DD_TAGS"
-	DockerHost                                           = "DOCKER_HOST"
-
-	DDOTLPgRPCEndpoint = "DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT"
-	DDOTLPHTTPEndpoint = "DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT"
-
-	// KubernetesEnvvarName Env var used by the Datadog Agent container entrypoint
-	// to add kubelet config provider and listener
-	KubernetesEnvVar = "KUBERNETES"
 
-	ClusterChecksConfigProvider = "clusterchecks"
+	// keep in common? will this eventually be needed for dap?
```

Comment:
it's fine to move these too imo. if we end up separating crd and controller modules, probably we will revisit what stays in the module anyway.